src/config/index.js

```diff
@@ -182,6 +186,7 @@ let config = {
   components,
 };
 
+config = simpleLink(minimalDefault(applySlateConfiguration(config)));
```

Comment:
Why do we need this explicitly and not allow slate to set it by itself? As an add-on?